Dockerfile

```diff
@@ -41,6 +41,8 @@ RUN apt-get update -qq && \
         qemu-system-arm \
         qemu-system-mips \
         qemu-system-ppc \
+        qemu-system-riscv32 \
+        qemu-system-riscv64 \
```

Comment:
There are no such packages in Debian. The package that contains both `qemu-system-riscv32` and `qemu-system-riscv64` binaries is `qemu-system-misc`.

Reply:
Ahh, good catch, thank you